### ISSUE TITLE
Camera animation not respected in viewport

### ISSUE THREAD

**Issue by AlexSchwank** [PixarAnimationStudios/USD#997]
### Description of Issue
In the new 19.11 RC it seems there's an issue with animated cameras. The Xform values get updated but there's no change in the viewport when looking through the camera. Selecting a different renderer doesn't fix the issue.

```
#usda 1.0
(
    defaultPrim = "world"
    endTimeCode = 9
    startTimeCode = 1
    upAxis = "Y"
)

def "World"
{
    def Camera "main_cam"
    {
        float4[] clippingPlanes = []
        float2 clippingRange = (0.15815565, 6045.622)
        float focalLength = 50
        float focusDistance = 5
        float fStop = 5.6
        float horizontalAperture = 41.4214
        float horizontalApertureOffset = 0
        token projection = "perspective"
        float verticalAperture = 23.299536
        float verticalApertureOffset = 0

        double3 xformOp:translate.timeSamples = {
            1: (0, 0, 1),
            2: (0, 0, 2),
            3: (0, 0, 3),
            4: (0, 0, 4),
            5: (0, 0, 5),
            6: (0, 0, 6),
            7: (0, 0, 7),
            8: (0, 0, 8),
            9: (0, 0, 9),
        } 
        uniform token[] xformOpOrder = ["xformOp:translate"]
    }
    def Mesh "visibleCube"
    {
        double3 xformOp:translate = (0, 0, -10)
        uniform token[] xformOpOrder = ["xformOp:translate"]

        uniform bool doubleSided = 1
        float3[] extent = [(-0.5, -0.5, -0.5), (0.5, 0.5, 0.5)]
        int[] faceVertexCounts = [3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3, 3]
        int[] faceVertexIndices = [0, 1, 2, 2, 1, 3, 2, 3, 4, 4, 3, 5, 4, 5, 6, 6, 5, 7, 6, 7, 0, 0, 7, 1, 1, 7, 3, 3, 7, 5, 6, 0, 4, 4, 0, 2]
        point3f[] points = [(-0.5, -0.5, 0.5), (0.5, -0.5, 0.5), (-0.5, 0.5, 0.5), (0.5, 0.5, 0.5), (-0.5, 0.5, -0.5), (0.5, 0.5, -0.5), (-0.5, -0.5, -0.5), (0.5, -0.5, -0.5)]
        color3f[] primvars:displayColor = [(0.13320851, 0.13320851, 0.13320851)]
    }
    
}
```

### Steps to Reproduce
1. Open attached usd file with usdview
1. Make sure the camera is selected as active camera
1. Scrub frames -> no update in viewport but Xform value get's updated
1. Enable camera oracles and zoom out
1. Scrub frames -> see camera oracle moving

### Package Versions
19.11 RC

**Comment by poljere** [comment]
Hi Alex! Many thanks for reporting this issue. This is indeed a bug and we have been working on a fix for it, it will be available in GitHub soon.

**Comment by jtran56** [comment]
Filed as internal issue #USD-5625.

**Comment by sunyab** [comment]
This should be addressed by 1d0805434. Marking this as pending release.

**Comment by AlexSchwank** [comment]
Thanks for fixing this so quickly!

**Comment by sunyab** [comment]
This fix is in the 19.11 release, closing - thanks for the bug report!
